### DISCUSSION TITLE
Free form snake tags / way to add new tags

### DISCUSSION THREAD

**Discussion by TheApX** [BattlesnakeOfficial/feedback#125] [Feature Requests]
Set of tags that can be applied to battlesnakes is limited to a predefined set. While this set contains the most common tags for programming languages, algorithms and hosting platforms, some of them are missing, such as `Oracle Cloud` or `Genetic Algorithm`. But even when these are added, there will always be tags that aren't available but somebody would be willing to use them - players' creativity has no limits.

Why not allow players to add free form tags? Or if we want pre-moderation, why not create a way to suggest new tags?

-----

P.S. My personal list of missing tags:
* `Oracle Cloud`
* `Self-hosted` / `Hosted at home`
* `Genetic Algorithm`
* `GMO`
* `Non-GMO`

**Comment by coreyja** [comment]
I really like the `GMO` and `Non-GMO` tags lol

**Comment by bvanvugt** [comment]
Awesome, just added these tags now:
* Oracle Cloud
* Self-Hosted
* Genetic
* GMO
* Non-GMO

Re: Custom tags, yeah moderation is key. We can be very responsive adding new tags, both here and on Discord. Maybe we just need to communicate that better?

**Reply by TheApX** [reply]
Yes, when you add tags to a snake it is not clear at all how you can create new tags. A button there or at least a link to discord/here with a note would make it much clearer.
